packages/console/src/pages/SignInExperience/components/BrandingForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { BrandingStyle, SignInExperience } from '@logto/schemas';
+import React from 'react';
+import { Control, Controller, UseFormRegister, UseFormWatch } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import FormField from '@/components/FormField';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import Switch from '@/components/Switch';
+import TextInput from '@/components/TextInput';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  register: UseFormRegister<SignInExperience>;
+  control: Control<SignInExperience>;
+  watch: UseFormWatch<SignInExperience>;
+};
+
+const BrandingForm = ({ register, control, watch }: Props) => {
```

Comment:
We don't necessarily need to pass in these parameters from parent component. Try `useFormContext()`

Reply:
updated